helm/ffc-demo-web/redis-values.yaml

```diff
@@ -1,3 +1,6 @@
 cluster:
   enabled: false
+master:
+  persistence:
+    enabled: false
```

Comment:
Disabled persistence as we don't need any persistent volume claim for Redis in development. The helm install script kept hanging due to failing to find an available persistent volume.